src/app/modules/onboarding/module.nim

```diff
@@ -5,25 +5,59 @@ import view, controller
 
 import app/global/global_singleton
 import app/core/eventemitter
+import app_service/service/general/service as general_service
+import app_service/service/accounts/service as accounts_service
+import app_service/service/devices/service as devices_service
+import app_service/service/keycardV2/service as keycard_serviceV2
 
 export io_interface
 
 logScope:
   topics = "onboarding-module"
 
+type PrimaryFlow* {.pure} = enum
+  Unknown = 0,
+  CreateProfile,
+  Login
+
+type SecondaryFlow* {.pure} = enum
+  Unknown = 0,
+  CreateProfileWithPassword,
+  CreateProfileWithSeedphrase,
+  CreateProfileWithKeycard,
+  CreateProfileWithKeycardNewSeedphrase,
+  CreateProfileWithKeycardExistingSeedphrase,
+  LoginWithSeedphrase,
+  LoginWithSyncing,
+  LoginWithKeycard
```

Comment:
My idea was to use the C++ [enum](https://github.com/status-im/status-desktop/pull/16722/files#diff-7f10de1672b6e797b2fdc027c8aa3c439cdc341d6ec2614f6ea418cb6c12279d) here as the only source of truth (which I created in the other PR which is not merged yet ofc), and also in QML. Perhaps later :)

Reply:
Ah yeah. I didn't find an enum that was usable, so I created one, but I have nothing against getting rid of this one to share the same you have once I integrate.
Do you know how to access the enum from Nim?

Reply:
> Do you know how to access the enum from Nim?

Not exactly but with my slightly limited NIM knowledge, it should be possible :laughing: